src/agent/onefuzz-agent/src/tasks/report/crash_report.rs

```diff
@@ -139,9 +146,57 @@ impl CrashTestResult {
         }
         Ok(())
     }
+
+    pub async fn save_regression(
+        self,
+        original_report: Option<CrashReport>,
+        reports: &Option<SyncedDir>,
+        no_repro: &Option<SyncedDir>,
+        prefix: impl AsRef<str>,
+    ) -> Result<()> {
+        match self {
+            Self::CrashReport(report) => {
+                // Use SHA-256 of call stack as dedupe key.
+                if let Some(reports) = reports {
+                    let name = format!("{}{}", prefix.as_ref(), report.unique_blob_name());
+                    if upload_or_save_local(
+                        &RegressionReport {
+                            crash_result: Self::CrashReport(report),
+                            original_report,
+                        },
+                        &name,
+                        reports,
+                    )
+                    .await?
+                    {
+                        event!(new_unique_report; EventData::Path = name);
```

Comment:
Is new_unique_report the correct event here?

Reply:
I think this and new_unable_to_reproduce might not be the right result here.  These are not new crashes, these are reproducing the existing crashes